packages/material-ui/src/RadioGroup/RadioGroup.test.js

```diff
@@ -175,60 +189,44 @@ describe('<RadioGroup />', () => {
   describe('prop: onChange', () => {
     it('should fire onChange', () => {
       const handleChange = spy();
-      const wrapper = shallow(
+      const wrapper = mount(
         <RadioGroup value="" onChange={handleChange}>
-          <Radio />
+          <Radio value="woofRadioGroup" />
           <Radio />
         </RadioGroup>,
       );
 
-      const internalRadio = wrapper.children().first();
-      const event = { target: { value: 'woofRadioGroup' } };
-      internalRadio.simulate('change', event, true);
+      const eventMock = 'something-to-match';
+      findRadioInput(wrapper, 'woofRadioGroup').simulate('change', { eventMock });
       assert.strictEqual(handleChange.callCount, 1);
-      assert.strictEqual(handleChange.calledWith(event), true);
+      assert.strictEqual(handleChange.calledWithMatch({ eventMock }), true);
     });
 
     it('should chain the onChange property', () => {
       const handleChange1 = spy();
       const handleChange2 = spy();
-      const wrapper = shallow(
+      const wrapper = mount(
         <RadioGroup value="" onChange={handleChange1}>
-          <Radio onChange={handleChange2} />
+          <Radio value="woofRadioGroup" onChange={handleChange2} />
           <Radio />
         </RadioGroup>,
       );
 
-      const internalRadio = wrapper.children().first();
-      internalRadio.simulate('change', { target: { value: 'woofRadioGroup' } }, true);
+      findRadioInput(wrapper, 'woofRadioGroup').simulate('change');
       assert.strictEqual(handleChange1.callCount, 1);
       assert.strictEqual(handleChange2.callCount, 1);
     });
   });
 
-  describe('register internal radios to this.radio', () => {
-    it('should add a child', () => {
-      const wrapper = mount(
-        <RadioGroup value="">
-          <Radio />
-        </RadioGroup>,
-      );
-      assert.strictEqual(wrapper.instance().radios.length, 1);
-    });
-
-    it('should keep radios empty', () => {
-      const wrapper = mount(<RadioGroup value="" />);
-      assert.strictEqual(wrapper.instance().radios.length, 0);
-    });
-  });
-
   describe('warnings', () => {
     beforeEach(() => {
       consoleErrorMock.spy();
+      stub(React, 'useEffect').callsFake(React.useLayoutEffect);
```

Comment:
Is this a substitute for the usage of `import { act } from 'react-dom/test-utils';`?

Reply:
Couldn't get it to work, but just figured it out.